pi4j-core/src/main/java/com/pi4j/runtime/impl/DefaultRuntime.java

```diff
@@ -70,7 +67,6 @@
     private final RuntimeProperties properties;
     private final List<Plugin> plugins = new ArrayList<>();
     private boolean isShutdown = false;
-    private final ExecutorService executor = Executors.newSingleThreadExecutor();
```

Comment:
The changes here seem to not be required for the JUnit changes, no? Should this be in a different pull request? Or why did you remove the shutting down?

Reply:
this slipped in while working (long time ago) on #18 and is still part of branch https://github.com/Pi4J/pi4j-v2/tree/feature/%2318. In that branch I also did the move to junit5 but extracted it into this separate branch so this could be merged independent of the work on the executorservice which actually still needs architectural investigation first.